src/instructlab/lab.py

```diff
@@ -825,23 +841,44 @@ def download(ctx, repository, release, filename, model_dir, hf_token):
         if ctx.obj is not None:
             hf_logging.set_verbosity(ctx.obj.config.general.log_level.upper())
         files = list_repo_files(repo_id=repository, token=hf_token)
+        local_dir = model_dir
         if any(".safetensors" in string for string in files):
             if not os.path.exists(os.path.join(model_dir, repository)):
                 os.makedirs(name=os.path.join(model_dir, repository), exist_ok=True)
+            local_dir = os.path.join(model_dir, repository)
             snapshot_download(
                 token=hf_token,
                 repo_id=repository,
                 revision=release,
-                local_dir=os.path.join(model_dir, repository),
+                local_dir=local_dir,
             )
         else:
             hf_hub_download(
                 token=hf_token,
                 repo_id=repository,
                 revision=release,
                 filename=filename,
-                local_dir=model_dir,
+                local_dir=local_dir,
             )
+
+        model_sigstore_bundle_path = f"{filename}.sigstore.json"
+        if model_sigstore_bundle_path in files:
```

Comment:
this will already be there is we do snapshot download, right? so this should be within the `else:` above?